pyproject.toml

```diff
@@ -31,6 +31,7 @@ dependencies = [
     "fastapi>=0.115.8",
     "setuptools>=75.8.0",
     "sse-starlette>=2.2.1",
+    "gitpython>=3.1.44",
```

Comment:
Remember we talked about NOT putting every package in pyproject? Let's try to use "dynamic packages"